src/logging/Logger.ts

```diff
@@ -79,30 +81,35 @@ export interface Logger extends SimpleLogger {
  * leaving only the implementation of {@link SimpleLogger}.
  */
 export abstract class BaseLogger implements Logger {
-  public abstract log(level: LogLevel, message: string): Logger;
+  public abstract log(level: LogLevel, message: string, meta?: any): Logger;
```

Comment:
Fun fact: all log functions accepted a meta object for almost 2 years until it was removed a month ago in #1248  because no logger actually ever used it.

Seeing as we know exactly what the type is of `meta`, and that is only what loggers can interpret, I would use an explicit type for that.